src/nvim/testdir/test_normal.vim

```diff
@@ -1226,29 +1226,6 @@ func Test_normal23_K()
   set iskeyword-=%
   set iskeyword-=\|
 
-  " Only expect "man" to work on Unix
```

Comment:
Avoid deleting lines in the oldtests. It makes merging upstream changes much easier if the lines exist, but are either commented-out (for 1-2 lines) or wrapped in a guard (many lines, such as this case).

To avoid indenting the skipped lines (again to avoid merge-conflicts from upstream), we can do an early-return here:

```
if has('nvim')  " Nvim K uses :terminal. #15398
  let &keywordprg = k
  bw!
endif
```